components/skus/browser/rs/cxx/src/lib.rs

```diff
@@ -192,10 +192,14 @@ pub struct CppSDK {
 }
 
 fn initialize_sdk(ctx: UniquePtr<ffi::SkusContext>, env: String) -> Box<CppSDK> {
-    tracing_subscriber::fmt()
+    match tracing_subscriber::fmt()
         .event_format(log::CppFormatter::new())
         .with_max_level(tracing::Level::TRACE)
-        .init();
+        .try_init()
+    {
+        Ok(_) => println!("tracing_subscriber - init success"),
```

Comment:
@bsclifton I'm curious this handling is fine. This was added to avoid unused result warning.

Reply:
ah interesting; I think it's ok
cc: @evq